addon/components/ember-youtube.js

```diff
@@ -325,59 +341,73 @@ export default Component.extend({
 
 	// OK, this is stupid but couldn't access the "event" inside
 	// an ember action so here's a manual click handler instead.
-	progressBarClick() {
+	addProgressBarClickHandler() {
+		this.element.addEventListener(
+			"click",
+			this.progressBarClick.bind(this),
+			false
+		);
+	},
+	progressBarClick(event) {
 		let self = this;
-		this.$().on('click', 'progress', function (event) {
-			// get the x position of the click inside our progress el
-			let x = event.pageX - $(this).position().left;
-			// convert it to a value relative to the duration (max)
-			let clickedValue = x * this.max / this.offsetWidth;
-			// 250 = 0.25 seconds into player
-			self.set("currentTime", clickedValue);
-			self.send('seekTo', clickedValue);
-		});
+		let element = event.srcElement;
+		if (element.tagName.toLowerCase() !== "progress") return;
+		// get the x position of the click inside our progress el
+		let x = event.pageX - element.getBoundingClientRect().x;
+		// convert it to a value relative to the duration (max)
+		let clickedValue = (x * element.max) / element.offsetWidth;
+		// 250 = 0.25 seconds into player
+		self.set("currentTime", clickedValue);
+		self.send("seekTo", clickedValue);
+	},
+	removeProgressBarClickHandler() {
```

Comment:
the event handler wasn't removed previously, so now it is cleaned up when the component is removed from the DOM.